app/templates/documents.rb

```diff
@@ -2,8 +2,8 @@
 
 class Documents
   include Templatable
-  LIGHT_STEMMERS = { fr: "french", de: "german", es: "spanish", it: "italian", pt: "portuguese" }
-  STANDARD_STEMMERS = { en: "english", fi: "finnish", hu: "hungarian", ru: "russian", sv: "swedish" }
+  LIGHT_STEMMERS = { fr: 'french', de: 'german', es: 'spanish', it: 'italian', pt: 'portuguese' }
+  STANDARD_STEMMERS = { en: 'english', fi: 'finnish', hu: 'hungarian', ru: 'russian', sv: 'swedish', hi: 'hindi', bn: 'bengali' }
```

Comment:
Rubocop also flagged these lines as "Freeze mutable objects assigned to constants," but I'm not sure if we want to do that here.